python/uperf/uperf_plugin.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3.9
+
+import sys
+import typing
+from dataclasses import dataclass
+from typing import List
+from arcaflow_plugin_sdk import plugin
+import subprocess
+import os
+import re
+import enum
+
+@dataclass
+class UPerfServerParams():
+    # How long to run the server
+    run_duration: int = 60
+
+@dataclass
+class UPerfServerError():
+    error_code: int
+    error: str
+
+@dataclass
+class UPerfServerResults():
+    pass
+
+@dataclass
+class IProtocol(enum.Enum):
+    TCP = "tcp"
+    UDP = "udp"
+
+@dataclass
+class UPerfParams:
+    """
+    This is the data structure for the input parameters of the step defined below.
+    """
+    server_addr: str = "127.0.0.1"
```

Comment:
Input schema params need metadata documentation (name and description)

Reply:
It looks like we also have some uperf parameter support to add still for MVP. And I wonder if we should consider adding an IP address data type to the SDK.

Reply:
Documentation done. The address should stay a string, since it could be an ip address, domain, or hostname. For example, in the container, it uses "server" for the addr, which is defined in the docker-compose.yml as the cluster network.